src/Compilers/CSharp/Test/Semantic/Semantics/RefFieldTests.cs

```diff
@@ -12889,6 +13034,122 @@ class Program
             // https://github.com/dotnet/roslyn/issues/62780: Test additional cases with [UnscopedRef].
         }
 
+        [Fact]
+        public void RefAssignMemberOfReturnOnlyParameter()
+        {
+            var source = @"
+using System.Diagnostics.CodeAnalysis;
+
+ref struct RS1
+{
+    public int I1;
+}
+
+ref struct RS2
+{
+    public ref int I2;
+}
+
+class Program
+{
+    static void M([UnscopedRef] ref RS1 rs, ref RS2 rs2)
```

Comment:
>[UnscopedRef]  [](http://example.com/codeflow?start=18&length=14)

Minor: Looks like `[UnscopedRef]` can be removed to remove an unnecessary error, here and in test below.